supervisors/sensitivity_analysis_morris/scalarmapi.R

```diff
@@ -0,0 +1,106 @@
+setGeneric("check_repetitions", function(.Object, params) {
+  .Object
+})
+setGeneric("schedule_point", function(.Object, params) {
+  .Object
+})
+setGeneric("get_result", function(.Object, params) {
+  .Object
+})
+setGeneric("mark_as_complete", function(.Object, result) {
+  .Object
+})
+Scalarm = setClass("Scalarm",
```

Comment:
@Zuchens - nie każę teraz tego poprawiać, ale formatowanie kodu w pliku jest niespójne i niezgodne z "naszymi standardami". Przykładowo "=" czasami jest przyklejone do lewej i prawej, czasami tylko do prawej itd...
Chyba, że jest jakiś autoformatter kodu R to można użyć.
Styl kodowania stosujemy podobny do tego: https://github.com/bbatsov/ruby-style-guide